src/NodeServices/IAnalyticsClient.cs

```diff
@@ -312,6 +317,56 @@ public enum Actions
         /// </summary>
         GroupRemovedFrom,
 
+        /// <summary>
+        /// CreateGroup event, when a group is created
+        /// </summary>
+        CreateGroup,
+
+        /// <summary>
+        /// Preview event, when a node is Previewed
+        /// </summary>
+        Preview,
+
+        /// <summary>
+        /// Freeze event, when a node is Freezed
+        /// </summary>
+        Freeze,
+
+        /// <summary>
+        /// Rename event, when a node is Renamed
+        /// </summary>
+        Rename,
+
+        /// <summary>
+        /// ShowLabels event, when a node is set to Show Labels
+        /// </summary>
+        ShowLabels,
+
+        /// <summary>
+        /// ShowHelp event, when user wants to display help info.
+        /// </summary>
+        ShowHelp,
+
+        /// <summary>
+        /// SetLacing event, when user wants to set a lacing type.
+        /// </summary>
+        SetLacing,
+
+        /// <summary>
+        /// DismissedAlerts event, when alerts are dismissed.
+        /// </summary>
+        DismissedAlerts,
+
+        /// <summary>
+        /// SetAsInput event, when user wants to set a node as input.
+        /// </summary>
+        SetAsInput,
+
+        /// <summary>
+        /// SetAsOutput event, when user wants to set a node as output.
+        /// </summary>
+        SetAsOutput,
```

Comment:
hi @zeusongit While working on analytics as well I notice that 
1. we are adding more specific info in action name while same info could appear as category name already. e.g. group
2. we are putting description as empty in a lot of cases

Can we somehow make the actions more generic so they can be shared between different categories and rely on description to distinguish them?

Reply:
I had the same concern @QilongTang , but in this case we are tracking something pretty specific, like the menu items in the node context menu, so I had to specify exactly which menu item, it will be hard to make this generic, however I try to choose a generic action item wherever possible.
In this case I can make the following changes to use more generic actions:

- Use `Create` instead of `CreateGroup`
- Use `Output`/`Input` instead of `SetAsOutput`/`SetAsInput`
- Use `Help`, `Labels`, `Lacing` instead of `ShowHelp`, `ShowLabels`, `SetLacing`
- `Freeze`, `Preview`, `Rename` are pretty generic already.

Let me know if these seem appropriate to you.